### PR TITLE
run feature test suite on IE9 on master CI with Sauce

### LINKED ISSUE #54: Enable Saucelabs for ongoing UI tests
https://github.com/department-of-veterans-affairs/caseflow/issues/550

### PR DESCRIPTION
**Pull request by shanear**
fixes #54

here's a working run in CI
https://travis-ci.org/department-of-veterans-affairs/caseflow-certification/builds/117986741

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +27,7 @@ script:
 bundler_args: "--without development staging production"
 cache:
   directories:
+    - travis-phantomjs
```
Comment (orischwartz) [inline .travis.yml]: Is this an attempt to fix #113 ? I think we'd also need to do a bash conditional -- do not wget if exists.

Comment (shanear) [inline .travis.yml]: yep, i've found that out. but i suck at bash, so i haven't added that part :cry:.

Comment (shanear) [inline .travis.yml]: alright i took that out, we can do the caching in a seperate pr

**Comment by paultag** [comment]
This broke prod :(

Running asset precompile wound up neededing rspec, which broke the precompile.

Thoughts on a better way to do this, @shanear ? Can we require conditionally, if this rake job is being run?

**Comment by paultag** [comment]
https://www.relishapp.com/rspec/rspec-core/docs/command-line/rake-task

**Comment by shanear** [comment]
doh, didn't even think about that. yeah, that's what we should be doing

**Pull request merged**